src/openrct2/peep/Guest.cpp

```diff
@@ -3881,8 +3883,9 @@ void Guest::UpdateRideFreeVehicleEnterRide(Ride* ride)
 
     if (PeepFlags & PEEP_FLAGS_TRACKING)
     {
-        auto nameArgLen = FormatNameTo(gCommonFormatArgs);
-        ride->FormatNameTo(gCommonFormatArgs + nameArgLen);
+        auto ft = Formatter::Common();
+        FormatNameTo(ft);
+        ride->FormatNameTo(gCommonFormatArgs + ft.NumBytes());
```

Comment:
As ride format name to takes a formatter you can pass it the formatter. No need to mess with mum bytes or the raw pointer gcommonformatargs. It already handles where to put the arguments safely.

`ride->FormatNameTo(ft);`